config/initializers/swagger_blocks.rb

```diff
@@ -0,0 +1,41 @@
+require "swagger/blocks"
+
+module ApiFlashcards
+  include Swagger::Blocks
+  host = if ENV['RAILS_ENV'] == "production"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.